addons/website_rating/models/mail_message.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+
+class MailMessage(models.Model):
+    _inherit = 'mail.message'
+
+    rating_value = fields.Float("Rating Value", compute='_compute_rating_value', search='_search_rating_value')
+
+    @api.multi
+    def _compute_rating_value(self):
```

Comment:
Is the rating value the last given value, am I right ? A small comment could help here I think.